### ISSUE TITLE
FF 55.0.3

### ISSUE THREAD

**Issue by keypey** [veto64/JavaScript-Whitelist#1]
Hi.
I am yours add-on user. But on: Wndows10-32Bit/FF 55.0.3 there are no add-on icon.

Can You check why?
--
Thanks, 
kp.,

**Comment by veto64** [comment]
i just checked this add-on with a windows server 2013 32Bit/FF 55.0.3 and i got the icon.
the add-on can not control the positon of its menu icon. maybe the icon is hidden under the menu button icon. try to click the 3 bar menu icon, than customize + button, after you should be able to arange most
![customize_icons](https://user-images.githubusercontent.com/6167461/29828655-9bc3c13e-8d07-11e7-9e54-5faf15433792.png)
 menu icons.

**Comment by keypey** [comment]
I check my two W10-32/bit.
[https://drive.google.com/open?id=0B6BFQ0vlLSkHNGo1dEZ1enRFOFE](url)

Unfortunately, I can not find it.
--
kp.,

**Comment by keypey** [comment]
It works. On two separate workstations (W10-32 / bit). On each separately, I removed all the add-ons. Then I turned them on in turn. I was spending every time there was a conflict of additive with a whitelist. And I found. IE Tab + v: 2.04.20110724.1-signed.1-signed After installing this add-on (without using it), "whitelist" stopped working.
So.
Removed IE Tab +, and everything is ok.

Sorry for the confusion,

--
kp.,
